src/SprykerEco/Zed/Econda/Business/Exporter/Runner.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Exporter;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Shared\Kernel\Store;
+use SprykerEco\Zed\Econda\Business\Exporter\Exception\BatchResultException;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Runner implements RunnerInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Econda\Business\Exporter\ExporterInterface
+     */
+    protected $exporter;
+
+    /**
+     * @var \SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleInterface
+     */
+    protected $localeFacade;
+
+    /**
+     * @param \SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleInterface $localeFacade
+     * @param \SprykerEco\Zed\Econda\Business\Exporter\ExporterInterface $exporter
+     */
+    public function __construct(
+        EcondaToLocaleInterface $localeFacade,
+        ExporterInterface $exporter
+    ) {
+        $this->localeFacade = $localeFacade;
+        $this->exporter = $exporter;
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return array
+     */
+    public function runExport(OutputInterface $output)
+    {
+        $storeCollection = Store::getInstance()->getAllowedStores();
```

Comment:
I would suggest to have `EcondaToStoreBridge` and use it

Reply:
No Bridge here, just store has to be initialised in dependency provider.

Reply:
@alex-galych your comment means that you like it? :)

Reply:
@gerner-spryker please advice, what should I change here?